scripts/update/0.5-0.6/update.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+
+kubectl create ns update-host-files --dry-run=client -o yaml | kubectl apply -f -
+kubectl create -n update-host-files cm script --from-file "${script_dir}"/cluster.sh --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
same as above

Reply:
There is no easy way to update the the configmap contents without having to delete, recreate or fiddle with the contents of configmap. If we want to maintain a shell script instead of configmap yaml file this is the easiest way to do it.